cmd/cmd.go

```diff
@@ -99,6 +100,7 @@ func initConfig() {
 			fmt.Fprintln(os.Stderr, err)
 		}
 	}
+	fmt.Println("Using config file:", viper.ConfigFileUsed())
```

Comment:
1. Unused constant declaration on line 31, it should be removed.

2. The function name should be capitalized to follow Go naming convention, from `initConfig` to `InitConfig`.

3. The `fmt.Println` statements for home directory and config file should be removed since they are not necessary and might cause unexpected output when running the program.

4. The error handling on line 88 only prints to stderr, it would be better to return the error to the caller or log the error to a file.

5. It is recommended to add error handling for the `viper.ReadInConfig()` function on line 94 to handle cases where the config file is not found or cannot be read.

6. It is also recommended to add input validation for the `cfgFile` variable to prevent directory traversal attacks or other security vulnerabilities.